rct229/rulesets/ashrae9012019/section5/section5rule30.py

```diff
@@ -124,6 +125,12 @@ def rule_check(self, context, calc_vals=None, data=None):
                     == ABSORPTION_THERMAL_EXTERIOR
                 )
 
+            def is_tolerance_fail(self, context, calc_vals=None, data=None):
+                return std_equal(
+                    calc_vals["absorptance_thermal_exterior_p"],
+                    ABSORPTION_THERMAL_EXTERIOR,
+                )
+
```

Comment:
Move this function after the `get_pass_msg`.
In evaluate function, this function is called after `get_pass_msg` function.